### ISSUE TITLE
Error run the Create Property

### ISSUE THREAD

**Issue by lotusccong** [SemanticMediaWiki/SemanticBundle#29]
After installed Semantic Bundle 4.0.0 in Mediawiki 1.36.1 , Page forms 4.9.5  ,  PHP 7.4.19, MySQL 5.7.34 ,.
I encounter this error to create the property from Page Forms - Create A property. The rest of the function like create class, category, form, and template are working fine.

[58f86fddabf3b13f31029120] /demo/index.php/Special:CreateProperty Error: Call to a member function getDatatypeLabels() on null

Backtrace:

from /home/sjkcyuhu/public_html/tbpedia.org/demo/extensions/PageForms/specials/PF_CreateProperty.php(99)
#0 /home/sjkcyuhu/public_html/tbpedia.org/demo/extensions/PageForms/specials/PF_CreateProperty.php(22): PFCreateProperty->printCreatePropertyForm(NULL)
#1 /home/sjkcyuhu/public_html/tbpedia.org/demo/includes/specialpage/SpecialPage.php(646): PFCreateProperty->execute(NULL)
#2 /home/sjkcyuhu/public_html/tbpedia.org/demo/includes/specialpage/SpecialPageFactory.php(1386): SpecialPage->run(NULL)
#3 /home/sjkcyuhu/public_html/tbpedia.org/demo/includes/MediaWiki.php(309): MediaWiki\SpecialPage\SpecialPageFactory->executePath(Title, RequestContext)
#4 /home/sjkcyuhu/public_html/tbpedia.org/demo/includes/MediaWiki.php(913): MediaWiki->performRequest()
#5 /home/sjkcyuhu/public_html/tbpedia.org/demo/includes/MediaWiki.php(546): MediaWiki->main()
#6 /home/sjkcyuhu/public_html/tbpedia.org/demo/index.php(53): MediaWiki->run()
#7 /home/sjkcyuhu/public_html/tbpedia.org/demo/index.php(46): wfIndexMain()
#8 {main}

**Comment by kghbln** [comment]
Thanks for reporting. Could you try to use master of the bundle and see if the error goes away?

**Comment by lotusccong** [comment]
Hi Kastern,What you mean use the master of the bundle?Need reinstall the Semantic bundle?On 28 Jun 2021 5:17 pm, Karsten Hoffmeyer ***@***.***> wrote:
Thanks for reporting. Could you try to use master of the bundle and see if the error goes away?

**Comment by TSchroth** [comment]
HI, 
I am experiencing issue with both Create Class and Create Property. I installed SemanticBundle 4.0.0, SMW 3.2.3, PF 4.9.5 in MediaWiki 1.35.2 PHP:7.4.20 on June 8th this year. I am more than happy to install Master, if you give me a brief pointer to where I can learn how to do it. (is it @dev instead of ~4.0 in the composer.local.json like in SMW?)

CreateClass
```
[5a1badcfc7660fcdc9b944a5] /index.php/Special:CreateClass Error from line 219 of /var/www/wiki-name.de/extensions/PageForms/specials/PF_CreateClass.php: Call to a member function getDatatypeLabels() on null

Backtrace:

#0 /var/www/wiki-name.de/includes/specialpage/SpecialPage.php(600): PFCreateClass->execute()
#1 /var/www/wiki-name.de/includes/specialpage/SpecialPageFactory.php(635): SpecialPage->run()
#2 /var/www/wiki-name.de/includes/MediaWiki.php(307): MediaWiki\SpecialPage\SpecialPageFactory->executePath()
#3 /var/www/wiki-name.de/includes/MediaWiki.php(940): MediaWiki->performRequest()
#4 /var/www/wiki-name.de/includes/MediaWiki.php(543): MediaWiki->main()
#5 /var/www/wiki-name.de/index.php(53): MediaWiki->run()
#6 /var/www/wiki-name.de/index.php(46): wfIndexMain()
#7 {main}
```
CreateProperty
```

[2ef9abee990dd3adb1b9fe98] /index.php/Special:CreateProperty Error from line 99 of /var/www/wiki-name.de/extensions/PageForms/specials/PF_CreateProperty.php: Call to a member function getDatatypeLabels() on null

Backtrace:

#0 /var/www/wiki-name.de/extensions/PageForms/specials/PF_CreateProperty.php(22): PFCreateProperty->printCreatePropertyForm()
#1 /var/www/wiki-name.de/includes/specialpage/SpecialPage.php(600): PFCreateProperty->execute()
#2 /var/www/wiki-name.de/includes/specialpage/SpecialPageFactory.php(635): SpecialPage->run()
#3 /var/www/wiki-name.de/includes/MediaWiki.php(307): MediaWiki\SpecialPage\SpecialPageFactory->executePath()
#4 /var/www/wiki-name.de/includes/MediaWiki.php(940): MediaWiki->performRequest()
#5 /var/www/wiki-name.de/includes/MediaWiki.php(543): MediaWiki->main()
#6 /var/www/wiki-name.de/index.php(53): MediaWiki->run()
#7 /var/www/wiki-name.de/index.php(46): wfIndexMain()
#8 {main}

```
thank you very much

**Comment by lotusccong** [comment]
After I upgrade the PageForms from 4.X to 5.2.1 . There is no error to create the Property anymore.

**Comment by JeroenDeDauw** [comment]
Released Semantic Bundle 5.0.0 which uses Page Forms 5.x
